### PR TITLE
Migrated from cargo-make to cargo-vita

### PR DESCRIPTION
**Pull request by nikarh**

### CODE REVIEW

File: examples/std-hello-world/Cargo.toml [hunk 1]
```diff
@@ -8,3 +8,11 @@ publish = false
 # Change the following line to require the current version of vitasdk-sys
 vitasdk-sys = { path = "../.." }
 rand = "0.8.5"
+
+[profile.release]
+# lto = true
```
Comment (zetanumbers) [inline examples/std-hello-world/Cargo.toml]: `[profile]` field is ignored if not in the workspace root
```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 /target
+examples/*/target
```
Comment (zetanumbers) [inline .gitignore]: Not needed.

```suggestion
```

File: Cargo.lock
Comment (zetanumbers) [inline Cargo.lock:1]: This isn't right. Please run `cargo fetch` and `cargo update` on  latest commit.

File: examples/std-hello-world/README.md [hunk 3]
```diff
@@ -18,29 +18,37 @@ To install the nightly toolchain with rust-src run:
...
+
+```
+cargo vita build vpk --upload --release
+```
+
+Or update the `eboot.bin` of an already installed vpk and run it:
+
+```
+cargo vita build eboot --update --run --release
+```
```
Comment (zetanumbers) [inline examples/std-hello-world/README.md]: Explain how to start ftp server on vita via vitashell and vitacompanion. Mention that `--run` flag would require vitacompanion.

**Review by zetanumbers — COMMENTED** [review]
I didn't try to run it on my end yet.

File: examples/std-hello-world/README.md (hunk 3 above)
Comment (nikarh) [reply examples/std-hello-world/README.md]: I'm not sure we should duplicate docs in every repo, I would rather have all these details here - https://github.com/vita-rust/cargo-vita (I've added link to readme)

File: .gitignore (hunk 2 above)
Comment (nikarh) [reply .gitignore]: Then we need to change `/target` to `target`, without it:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        examples/std-hello-world/target/

```

File: examples/std-hello-world/README.md (hunk 3 above)
Comment (nikarh) [reply examples/std-hello-world/README.md]: On the other hand it won't hurt also having it here. Done

**Review by zetanumbers — APPROVED** [review]
runs as expected

**Review by pheki — APPROVED** [review]

**Pull request merged**